kiosk/Readme.md

```diff
@@ -0,0 +1,19 @@
+# PlayOS Kiosk Browser
+
+Opens a website in a full screen, locked down browser based on [QtWebView](http://doc.qt.io/qt-5/qtwebview-index.html).
```

Comment:
Small mixup. Kiosk Browser uses QtWebEngine (http://doc.qt.io/qt-5/qtwebengine-index.html) not QtWebKit.